### DISCUSSION TITLE
Office_S_Tips/pki/2019/09/19/e-podpis_kilka_cytatow_z_aktow_prawnych

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#18] [Komentarze]
# Office_S_Tips/pki/2019/09/19/e-podpis_kilka_cytatow_z_aktow_prawnych

Ustawa z 05.09.2016 o usługach zaufania oraz identyfikacji elektronicznej

https://andrzejq.github.io/Office_S_Tips/pki/2019/09/19/e-podpis_kilka_cytatow_z_aktow_prawnych.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
